src/main/java/frc/robot/Config.java

```diff
@@ -255,6 +255,9 @@ public static final class AutoConstants {
         new TrapezoidProfile.Constraints(
             kMaxAngularSpeedRadiansPerSecond, kMaxAngularSpeedRadiansPerSecondSquared);
   }
+   public static final class BlingConstants {
+    public static int CANDLE = 15;
```

Comment:
You can put this line inside class CANID.
(Note: so far we always set CANID of 15 for CANDLE for Beetle and Poseidon.)

Reply:
After discussion at the meeting on Wednesday we have decided to keep the structure of having each mechanism have its own class, where the CANIDs (and any other constants) will live.